packages/core/src/Table/Table.js

```diff
@@ -1,56 +1,59 @@
 import propTypes from '@dhis2/prop-types'
-import React from 'react'
-import css from 'styled-jsx/css'
-import { Provider } from './TableContext.js'
-
-const tableStyles = css`
-    table {
-        border: 1px solid #e8edf2;
-        background-color: #ffffff;
-        min-width: 100%;
-        text-align: left;
-        border-collapse: collapse;
-        vertical-align: top;
-    }
-`
+import { colors } from '@dhis2/ui-constants'
+import React, { forwardRef } from 'react'
 
 /**
  * @module
  * @param {Table.PropTypes} props
  * @returns {React.Component}
- * @example import { Table } from '@dhis2/ui-core'
- * @see Live demo: {@link /demo/?path=/story/table--static-layout|Storybook}
+ * @example import { Table } from '@dhis2/ui'
+ * @see Specification: {@link https://github.com/dhis2/design-system/blob/master/organisms/data-table.md|Design system}
+ * @see Live demo: {@link /demo/?path=/story/table--default|Storybook}
  */
-export const Table = ({
-    children,
-    className,
-    dataTest,
-    suppressZebraStriping,
-}) => (
-    <Provider value={{ suppressZebraStriping }}>
-        <table className={className} data-test={dataTest}>
-            {children}
-
-            <style jsx>{tableStyles}</style>
-        </table>
-    </Provider>
+export const Table = forwardRef(
+    ({ children, className, dataTest, layout, width }, ref) => {
+        return (
+            <table className={className} data-test={dataTest} ref={ref}>
+                {children}
+                <style jsx>{`
+                    table {
+                        table-layout: ${layout};
+                        border-collapse: separate;
```

Comment:
Styling a table with collapsed borders is much easier. We've had to stick with separate borders because sticky cells don't work very well with collapsed borders